roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
     docker_options: >
       --insecure-registry={{ openshift.docker.hosted_registry_network }}
       {{ openshift.docker.options | default ('') }}
-  when: openshift.docker.hosted_registry_insecure | default(False) | bool
+  when: openshift.docker.hosted_registry_insecure | default(False) | bool and openshift.docker.hosted_registry_network is defined
```

Comment:
Shouldn't `openshift.docker.hosted_registry_network` always be set? We may need to make sure that the roles (master, node, etcd) that need it are getting a default set within `openshift_facts` lib.